scripts/gibbs.py

```diff
@@ -0,0 +1,303 @@
+# https://raw.githubusercontent.com/dipy/dipy/160b202e7cd0ed821e5e1bfc4cf14815700430f3/dipy/denoise/gibbs.py
```

Comment:
DIPY's code keeps progressing. Hence, we copied over this small function from them to be compatible with our multi-processed Gibb's unringing.
Reference: Tashrif's issue: https://github.com/dipy/dipy/issues/2757